projects/m1/001-area-of-a-room/solutions/SanchezLuca/solution_001-Area-of-a-room.py

```diff
@@ -0,0 +1,5 @@
+print('--Hello! Here you can calculate the area of a room in m^2.--')
+Width = float(input('Type the width in meter--> '))
```

Comment:
Attenzione! I nomi delle variabili dovrebbero iniziare in minuscolo e seguire una naming convention: https://en.wikipedia.org/wiki/Naming_convention_(programming)